cassandra/src/main/scala/akka/stream/alpakka/cassandra/package.scala

```diff
@@ -7,8 +7,8 @@ import com.google.common.util.concurrent.{ FutureCallback, Futures, ListenableFu
 
 import scala.concurrent.{ Future, Promise }
 
-package object cassandra {
-  implicit final class GuavaFutureOpts[A](val guavaFut: ListenableFuture[A]) extends AnyVal {
+package object GuavaFutureOpts {
```

Comment:
With this name it is no longer a package object, just a normal object, so remove that and make it package private for cassandra with `private[cassandra]` which will cover the class inside it as well. You will have to import the implicit class into the scope where it is used though, so that will be an `import GuavaFutureOpts._` in the classes you call `asScala`